### PR TITLE
SM: Added exports with labels blog post

### LINKED ISSUE #46: Add blog on exports with labels
Add blog post on exports with labels: https://docs.google.com/a/ona.io/document/d/1pG0f-ecOCy1_8wUww4q5rAhb1YP2gXioWz3ARjMYToY/edit?usp=sharing

### PR DESCRIPTION
**Pull request by smusula**
closes #46

### CODE REVIEW

**Comment by rowo** [comment]
@smusula In the line, "The **name** column, in both the survey and choices worksheets, specifies the variable name that defines the data entry point."... what do you mean by "... defines the data entry point."?

File: _posts/2016-02-26-exports-with-labels.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- General
+tags: []
+author_id: 10
+status: published
+type: post
+published: true
+meta:
+---
+
+We have introduced a new feature that allows users to export their data using labels as the column headers instead of field names. This works for CSV, Excel and CSV Zip exports.
```
Comment (ukanga) [inline _posts/2016-02-26-exports-with-labels.md]: I think anywhere we have 'users' we should replace with 'you'. We are targeting the person reading the post not a third party.

File: _posts/2016-02-26-exports-with-labels.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Using [XLSForm syntax](http://xlsform.org/#basic-format) the **label** column contains the text of the question that the user sees in the form and the **name** column in both the survey and choices worksheets specifies the variable name that defines the data entry point. When you export data from Ona in CSV, Excel, or CSV Zip formats, under advanced export settings, you can choose to include the labels in your exports along with the variable names, or you can choose to include only labels in your exports (i.e., without the variable names).
```
Comment (ukanga) [inline _posts/2016-02-26-exports-with-labels.md]: I think '... To include only labels without the variable names'. Something alluding to "include labels" and "include labels only" being the available options.

File: _posts/2016-02-26-exports-with-labels.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+type: post
+published: true
+meta:
+---
+
+We have introduced a new feature that allows users to export their data using labels as the column headers instead of field names. This works for CSV, Excel and CSV Zip exports.
+
+Using [XLSForm syntax](http://xlsform.org/#basic-format) the **label** column contains the text of the question that the user sees in the form and the **name** column in both the survey and choices worksheets specifies the variable name that defines the data entry point. When you export data from Ona in CSV, Excel, or CSV Zip formats, under advanced export settings, you can choose to include the labels in your exports along with the variable names, or you can choose to include only labels in your exports (i.e., without the variable names).  
+
+This feature is helpful for instance when using other statistical software for advanced data analysis, because it minimizes the work of replacing your variable names with labels. It’s also helpful in that it’s no longer necessary to go back to the XLSForm to be reminded of the questions.
```
Comment (ukanga) [inline _posts/2016-02-26-exports-with-labels.md]: I feel this paragraph may need some rewrite, would probably have a proposal when behind a computer.

File: _posts/2016-02-26-exports-with-labels.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## How it works
+
+In your XLSForm, say you have a question for respondent male or female and the variable name for this question is hhh_gender while the label is Gender of the household head.  To include both variable name and label in the data export, go to the form’s overview page; under the export data section, click on Show advanced export options and select Include labels. See the screenshot below:
```
Comment (ukanga) [inline _posts/2016-02-26-exports-with-labels.md]: **Include labels**

**Comment by rowo** [comment]
@ukanga I dont know how to respond to your diff notes. But I rewrote some stuff already and can include your change and push those.

**Pull request merged**